content/en/docs/tasks/administer-cluster/out-of-resource.md

```diff
@@ -119,7 +119,10 @@ The `kubelet` has the following default hard eviction threshold:
 * `nodefs.available<10%`
 * `imagefs.available<15%`
 
-On a Linux node, the default value also includes `nodefs.inodesFree<5%`.
+On Linux, the following are also set:
+
+* `pid.available<10%`
```

Comment:
It might be worth emphasizing that this is compared against `RLIMIT_NPROC`. I'm suggesting this because you might think it was compared against `kernel.pid_max` or against the ceiling from the relevant `pids` cgroup controller.

Reply:
is the best way to do it to add a link to the [line 34](https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/administer-cluster/out-of-resource.md#user-content-eviction-signals:~:text=node.stats.rlimit.maxpid) of this file. I agree that table entry may not be descriptive enough to deduct which percentage is used.

Reply:
(BTW This isn't feedback that'd block a merge - we could leave out the emphasis I'm suggesting)

A hyperlink back to the earlier detail sounds reasonable.